esp-hal/src/i2s/master/mod.rs

```diff
@@ -107,6 +107,9 @@ use crate::{
     DriverMode,
 };
 
+//#[cfg(esp32)]
```

Comment:
I'm not a fan of the organization (camera shouldn't be under the i2s master IMHO) but either way, camera needs to be gated for esp32

Reply:
Yeah, it's actually the slave mode. `i2s/slave/camera.rs`

Reply:
Eh, just place it as `i2s/camera.rs`, we didn't put the display driver anywhere either.

Reply:
IIRC i had to put it under `master` as i had to reference a bunch of things from `master` which aren't `pub(crate)` (or `pub`). but i can check on that again once i know that the code works (so far testing with ESP32-CAM wasn't successful).

(and yes, the commented-out `cfg` was an oversight which i'll fix)